### DISCUSSION TITLE
typos in ustcthesis-doc?

### DISCUSSION THREAD

**Discussion by fjtcin** [ustctug/ustcthesis#323] [General]
<img width="637" alt="image" src="https://user-images.githubusercontent.com/96152068/146630927-097e1029-5713-4ffc-8ffd-57cb3dd543d6.png">
In the section-style setting part of "ustcthesis 使用说明", "english" should be changed into "arabic", and I can't find "例1", "例2".

**Comment by zepinglee** [comment]
Surely they are typos. Thanks!

**Reply by zepinglee** [reply]
Fixed in 88a9af9.
